js/viewer/annotation/annotation-svg.js

```diff
@@ -525,12 +550,15 @@ function AnnotationSVG(parent, id, imgWidth, imgHeight, scale, ignoreReferencePo
 
         var svg = document.createElementNS("http://www.w3.org/2000/svg", "svg");
 
+        yArr = [10000, 12000, 14000, 16000, 18000, 20000, 22000];
```

Comment:
What's the purpose of this?

Reply:
It was added when I was testing some static creation of SVGs. Removed it.